velox/substrait/CMakeLists.txt

```diff
@@ -41,8 +41,18 @@ add_custom_command(
 add_custom_target(substrait_proto ALL DEPENDS ${PROTO_OUTPUT_FILES})
 add_dependencies(substrait_proto protobuf::libprotobuf)
 
-set(SRCS ${PROTO_SRCS} SubstraitParser.cpp SubstraitToVeloxExpr.cpp
-         SubstraitToVeloxPlan.cpp TypeUtils.cpp)
+set(SRCS
+    ${PROTO_SRCS}
+    SubstraitParser.cpp
+    SubstraitToVeloxExpr.cpp
+    SubstraitToVeloxPlan.cpp
+    TypeUtils.cpp
+    GlobalCommonVariable.cpp
```

Comment:
This file no longer exists, right? Let's remove.

Reply:
remove it.